### ISSUE TITLE
Getting Segmentation fault: 11  when running ciphey

### ISSUE THREAD

**Issue by JLP04** [Ciphey/Ciphey#362]
**⚠️IMPORTANT⚠️ if you do not fill this out, we will automatically delete your issue. We will not help anyone that cannot fill out this template.**

- [✅] Have you read our [Wiki page "Common Issues & Their Solutions"?](https://github.com/Ciphey/Ciphey/wiki/Common-Issues-&-Their-Solutions)

**Describe the bug**
When I run ciphey, I get "Segmentation fault: 11". Note: my plaintext was supposed to be encoded using multiple ciphers and I did pass `-p ausearch.enable_nested=True`
**Plaintext**
**⚠️IMPORTANT⚠️ The below code is non-negotiable for "Ciphey didn't decrypt...." problems. If you do not tell us your plaintext, we will not help you.** 

```

GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG
```
**Version**
**⚠️IMPORTANT⚠️ We need this information because different environments will induce different bugs in Ciphey**
 - OS/Distro: macOS Catalina
 - Python version: 3.8.5
 - Ciphey versions: 5.3.2 0.2.9 0.3.23
 - Did you use Docker? No

**Verbose Output**
**⚠️IMPORTANT⚠️ Verbose output will tell us why it's not working the way we expected it to be.**
Run Ciphey with `ciphey -vvv` and paste the results into [Pastebin.com](https://pastebin.com) or a [GitHub Gist](https://gist.github.com/)
[Link to my verbose output](https://pastebin.com/1QwhpPRd)
**To Reproduce**
1. What input did you use? `ciphey -p ausearch.enable_nested=True -t "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG"`
2. What flags / arguments did you use? 
`-p ausearch.enable_nested=True -t "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG"`

**Expected behavior**
I expected it to decode my encrypted text. 
**Any other information?**

**Comment by bee-san** [comment]
This is related to #238 and #256 :( 
The workaround is to use Docker :cry:

**Comment by JLP04** [comment]
How do I pass `-p ausearch.enable_nested=True` to the docker version?

**Comment by JLP04** [comment]
> This is related to #238 and #256 :(
> The workaround is to use Docker 😢

How do I pass `-p ausearch.enable_nested=True` to the docker version?

**Comment by bee-san** [comment]
> > This is related to #238 and #256 :(
> > The workaround is to use Docker cry
> 
> How do I pass `-p ausearch.enable_nested=True` to the docker version?

Have you tried this? Does not that work? The Docker image should build from our master branch (if I recall).

If it doesn't work, let me know and I'll fix it :smile:

**Comment by JLP04** [comment]
I ran `docker run -it --rm remnux/ciphey "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG"` but I'm not sure how to run it through docker with nested encryption.

**Comment by SkeletalDemise** [comment]
Hello. Does running `docker run -it --rm remnux/ciphey "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG" -p ausearch.enable_nested=True` work?

**Comment by JLP04** [comment]
No

**Comment by SkeletalDemise** [comment]
What happens when you run it? It might be that Ciphey cannot decrypt your text.

**Comment by ArnaubHie** [comment]
What happen￼

**Comment by lukasgabriel** [comment]
Try this: (-t flag before the string, other arguments before the -t flag)
 `docker run -it --rm remnux/ciphey -p ausearch.enable_nested=True -t "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG" `

**Comment by JLP04** [comment]
Running `docker run -it --rm remnux/ciphey "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG" -p ausearch.enable_nested=True` or `docker run -it --rm remnux/ciphey -p ausearch.enable_nested=True -t "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG"` returns `Traceback (most recent call last):
  File "/usr/local/bin/ciphey", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/ciphey/ciphey.py", line 229, in main
    config.update_param(parent, name, value)
  File "/usr/local/lib/python3.8/site-packages/ciphey/iface/_config.py", line 125, in update_param
    if _fwd.registry.get_named(owner).getParams()[name].list:
KeyError: 'enable_nested'
`

**Comment by bee-san** [comment]
Hey! Thanks for the comment. Our docker image is maintained by REMnux, but we can edit it.

I'll try to fix this :)

**Comment by Cyclic3** [comment]
> Running `docker run -it --rm remnux/ciphey "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG" -p ausearch.enable_nested=True` or `docker run -it --rm remnux/ciphey -p ausearch.enable_nested=True -t "GOWOKGFOTRSERRAUVRFFBFCAAWWAWGTWWQGJFMWBTKFTJASFAEZTZWIMDZRYFPZRATMLGFYDROWRBWLXKGQHGQWQXWLKQYCQOMFCUVJFQVFQTUIECCGDKTZVVJEJBG"` returns `Traceback (most recent call last): File "/usr/local/bin/ciphey", line 8, in <module> sys.exit(main()) File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__ return self.main(*args, **kwargs) File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main rv = self.invoke(ctx) File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke return ctx.invoke(self.callback, **ctx.params) File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke return callback(*args, **kwargs) File "/usr/local/lib/python3.8/site-packages/ciphey/ciphey.py", line 229, in main config.update_param(parent, name, value) File "/usr/local/lib/python3.8/site-packages/ciphey/iface/_config.py", line 125, in update_param if _fwd.registry.get_named(owner).getParams()[name].list: KeyError: 'enable_nested' `

That error is because the `-p ausearch.enable_nested=True` is a very new flag, and remnux uses an old version of ciphey

**Comment by bee-san** [comment]
This is being fixed in #591
